apps/server/src/modules/h5p-editor/service/h5p-player.service.ts

```diff
@@ -0,0 +1,27 @@
+import { H5PPlayer, ITranslationFunction } from '@lumieducation/h5p-server';
+
+import { h5pConfig, h5pUrlGenerator } from './config/h5p-service-config';
+import { ContentStorage } from './contentStorage.service';
+import { Translator } from './h5p-translator.service';
+import { LibraryStorage } from './libraryStorage.service';
+
+export const H5PPlayerService = {
```

Comment:
Service zu Provider umbenennen und in ein provider Odner verschieben.

Reply:
Done